src/layout/Summary2/SummaryComponent2/ComponentSummary.tsx

```diff
@@ -22,6 +24,14 @@ interface ResolveComponentProps {
 export function ComponentSummary({ componentNode, summaryOverrides, isCompact }: ComponentSummaryProps) {
   const override = summaryOverrides?.find((override) => override.componentId === componentNode.item.id);
 
+  const summaryNode = useTaskStore((state) => state.summaryNode);
+
+  const isRequired = 'required' in componentNode.item;
```

Comment:
It should only be required if required is in item AND it's true.
```suggestion
  const isRequired = 'required' in componentNode.item && componentNode.item;
```